ssl/internal.h

```diff
@@ -3533,6 +3537,25 @@ struct ssl_ctx_st {
   SSL_SESSION *(*get_session_cb)(SSL *ssl, const uint8_t *data, int len,
                                  int *copy) = nullptr;
 
+  struct {
+    int sess_connect = 0;             // SSL new conn - started
+    int sess_connect_renegotiate = 0; // SSL reneg - requested
+    int sess_connect_good = 0;        // SSL new conne/reneg - finished
+    int sess_accept = 0;              // SSL new accept - started
+    int sess_accept_renegotiate = 0;  // SSL reneg - requested
+    int sess_accept_good = 0;         // SSL accept/reneg - finished
+    int sess_miss = 0;                // session lookup misses
```

Comment:
sess_miss is never updated, but I think we can down where we check the session cache.